.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "tabWidth": 2,
+  "useTabs": false
```

Comment:
can I be that devil that adds some suggestions (everyone has a flavour so I'd really like @EddieJaoudeCommunity/coreteam to contribute to get the ideal config we all like)

This file also does support a simpler file format other than JSON

```
semi: true
singleQuote: true
printWidth: 80
tabWidth: 2
useTabs: false
```

- semi true for semi-colon everywhere (this is the most egregious)
- single quote to ensure we use single quotes everywhere (AirBnB styleguide and others I believe)
- printWidth because I like a ruler of 80 because I'm old school.
- tabWidth as you where, yes - spaces over tabs is great @Jashpatel1 

https://youtu.be/SsoOG6ZeyUI

I've added useTabs to my configuration as I assume everyone changes their IDE which is foolish, so I've pinched a bit of your config too.

Reply:
I agree with this config you are suggesting @stemount 👍

Reply:
Sounds good 👍